### PR TITLE
Add kube-scheduler profiles

### PR DESCRIPTION
**Pull request by YuryLysov**
## Description

Add two kube-scheduler profiles: high-node-utilization and no-scoring.
high-node-utilization profile schedules pod to node with most resource utilization, no-scoring disables scoring to speed up scheduling process.

## Why do we need it, and what problem does it solve?

Allows users to use predefined scheduler profiles without having to define them themselves.

## What is the expected result?

KubeSchedulerConfiguration is deployed to master node of a cluster with three profiles: default-scheduler,  high-node-utilization, no-scoring, and used by kube-scheduler pod in kube-system namespace. Each of them can be used in pod `spec.schedulerName`

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: control-plane-manager
type: feature
summary: Added two kube-scheduler profiles: high-node-utilization and no-scoring.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @YuryLysov. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @YuryLysov. I've started the workflow for label 'e2e/run/aws'...

**Review by mortelumina98 — APPROVED** [review]

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -838,6 +838,33 @@ rm -r ./kubernetes ./etcd-backup.snapshot
...
 ### Логика работы
 
+#### Профили планировщика
+
+Есть два преднастроенных профиля планировщика:
+
+* `default-scheduler`: профиль по умолчанию, который распределяет поды на узлы с наименьшей загрузкой.
+* `high-node-utilization`: профиль, при котором поды размещаются на узлах с наибольшей загрузкой.
+
+> Чтобы задать профиль планировщика, используйте в манифесте пода параметр `spec.schedulerName`.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:848]:
```suggestion
Чтобы задать профиль планировщика, укажите его параметре `spec.schedulerName` манифеста пода.
```

**Review by z9r5 — APPROVED** [review]

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 2]
```diff
@@ -835,6 +835,33 @@ The full list of plugins is available in the [Kubernetes documentation](https://
 
 ### Working logic
 
+#### Scheduler profiles
+
+There are two predefined scheduler profiles:
+
+* `default-scheduler` — the default profile that distributes pods to nodes with the lowest load;
+* `high-node-utilization` — a profile that places pods on nodes with the highest load.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:843]:
```suggestion
* `default-scheduler`: The default profile that distributes pods to nodes with the lowest load;
* `high-node-utilization`: A profile that places pods on nodes with the highest load.
```

**Review by RomanenkoDenys — APPROVED** [review]

**Review by sprait — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Pull request merged**